GNUmakefile

```diff
@@ -355,13 +355,6 @@ TEST_PACKAGES_FAST = \
 	unique \
 	$(nil)
 
-# Assume this will go away before Go2, so only check minor version.
-ifeq ($(filter $(shell $(GO) env GOVERSION | cut -f 2 -d.), 16 17 18), )
-TEST_PACKAGES_FAST += crypto/internal/nistec/fiat
-else
-TEST_PACKAGES_FAST += crypto/elliptic/internal/fiat
-endif
-
```

Comment:
I removed these, since they're an internal package and the name keeps changing. In Go 1.24, the name seems to have changed to crypto/internal/fips140/nistec/fiat.